source/model/Database.py

```diff
@@ -0,0 +1,24 @@
+class Database():
```

Comment:
Based on [PEP 257](https://www.python.org/dev/peps/pep-0257/) we have placed the docstring as the first line of the function and not above.